activation/e2e/builds_atx_v2_test.go

```diff
@@ -55,37 +54,17 @@ func TestBuilder_SwitchesToBuildV2(t *testing.T) {
 	db := sql.InMemory()
 	cdb := datastore.NewCachedDB(db, logger)
 
-	syncer := activation.NewMocksyncer(ctrl)
-	syncer.EXPECT().RegisterForATXSynced().DoAndReturn(func() <-chan struct{} {
-		synced := make(chan struct{})
-		close(synced)
-		return synced
-	}).AnyTimes()
-
-	svc := grpcserver.NewPostService(logger)
-	svc.AllowConnections(true)
-	grpcCfg, cleanup := launchServer(t, svc)
-	t.Cleanup(cleanup)
+	opts := testPostSetupOpts(t)
+	svc := initializeIDs(t, db, goldenATX, []*signing.EdSigner{sig}, cfg, opts)
```

Comment:
See my other comment: it feels odd to have a "multi-init" function and then use it to just initialize a single identity.

If you want to get rid of some of the code duplication (which in tests in my opinion is fine to have) look at my previous comment 🙂